src/flb_config_map.c

```diff
@@ -163,7 +163,7 @@ static int translate_default_value(struct flb_config_map *map, char *val)
         entry->val.d_num = atof(val);
     }
     else if (map->type == FLB_CONFIG_MAP_SIZE) {
-        entry->val.s_num = flb_utils_size_to_bytes(val);
+        entry->val.i_num = flb_utils_size_to_bytes(val);
```

Comment:
Why did you change this one now?

Reply:
Uh, it's a mistake.

Reply:
Please revert this.

Reply:
I pushed a wrong commit, sorry, please help to review again.